jdaviz/core/helpers.py

```diff
@@ -273,96 +274,96 @@ def get_model_parameters(self, models=None, model_label=None, x=None, y=None):
+
+                See `jupyterlab-sidecar <https://github.com/jupyter-widgets/jupyterlab-sidecar>`_
+                for the most up-to-date options.
+
+
         title : str, optional
             The title of the sidecar tab.  Defaults to the name of the
             application; e.g., "specviz".
-        anchor : str
-            Where the tab should appear, by default on the right. Options are:
-            {sidecar_anchor_values}.
+
+            NOTE: Only applicable to a "sidecar" display.
 
         Returns
         -------
-        sidecar
-            The ``sidecar.Sidecar`` object used to create the tab.
+        sidecar or None
+            The ``sidecar.Sidecar`` object used to create the tab or window or
+            None if no sidecar was created
 
         Notes
         -----
-        If this method is called in the "classic" Jupyter notebook, the app will
-        appear inline, as only lab has a mechanism to have multiple tabs.
-        See Also
-        --------
-        show_in_new_tab
-        show_inline
+        If "sidecar" is requested in the "classic" Jupyter notebook, the app will appear inline,
+        as only lab has a mechanism to have multiple tabs.
         """
-        if 'title' not in kwargs:
-            kwargs['title'] = self.app.config
+        try:
+            if loc == "inline":
+                display(self.app)
 
-        scar = Sidecar(**kwargs)
-        with scar:
-            display(self.app)
+            elif loc.startswith('sidecar'):
+                if loc != 'sidecar':
+                    anchor = loc.split(':')[1]
+                    kwargs['anchor'] = anchor
 
-        return scar
-    show_in_sidecar.__doc__ = show_in_sidecar.__doc__.format(
-        sidecar_anchor_values=repr(Sidecar.anchor.values)[1:-1])
+                # If title unset, default to the viz config
+                if 'title' not in kwargs:
+                    kwargs['title'] = self.app.config
 
-    def show_in_new_tab(self, **kwargs):
-        """
-        Display the Jdaviz application in a new tab in JupyterLab.
+                scar = Sidecar(**kwargs)
+                with scar:
+                    display(self.app)
 
-        Additional keywords not listed here are passed into the
-        ``sidecar.Sidecar`` constructor. See
-        `jupyterlab-sidecar <https://github.com/jupyter-widgets/jupyterlab-sidecar>`_
-        for the most up-to-date options.
+                return scar
 
-        Parameters
-        ----------
-        title : str, optional
-            The title of the sidecar tab.  Defaults to the name of the
-            application; e.g., "specviz".
+            elif loc == "new browser tab":
+                raise NotImplementedError
 
-        Returns
-        -------
-        sidecar
-            The ``sidecar.Sidecar`` object used to create the tab.
+            elif loc == "popout":
+                raise NotImplementedError
```

Comment:
to be analogous to the sidecar anchors, I'd suggest something like "popout:window" and "popout:tab"... but since they're not implemented or even documented here, that could be easily changed later (I just at least wanted to leave a papertrail while its still fresh).

Reply:
I don't understand why these need special `elif`. Why not just let them go through the same logic as other invalid inputs, given they are not even documented?

Reply:
Agreed! I think if we end up implementing different popout locations, I'd be a big fan of the above. I mainly put this codeblock here just to remind us to add this later